fcrepo-configs/src/main/resources/config/file-s3/repository.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name" : "repo",
+    "jndiName" : "",
+    "workspaces" : {
+        "predefined" : ["default"],
+        "default" : "default",
+        "allowCreation" : true,
+        "cacheSize" : 10000
+    },
+    "storage" : {
+        "persistence": {
+            "type": "file",
+            "path" : "${fcrepo.object.directory:target/objects}"
+        },
+        "binaryStorage" : {
+            "type" : "s3",
+            "username" : "${aws.accessKeyId}",
```

Comment:
If this is a concern for AWS keys (which, if generated and managed properly, can be granted narrow access only to the specific S3 bucket needed), it's also a concern for the database credentials used in the mysql and postgres configs. IOW, the issue isn't specific to whether this PR should be merged as-is. The code in this PR is consistent with current practice – passing everything through JAVA_OPTS. If that's not a good practice, that's probably best handled by a larger, top-level issue that also changes the handling of other secrets at the same time.